gherkin-utils/javascript/src/pretty.ts

```diff
@@ -1,29 +1,31 @@
 import * as messages from '@cucumber/messages'
 
-export default function pretty(gherkinDocument: messages.GherkinDocument): string {
+type Syntax = 'markdown' | 'gherkin'
+
+export default function pretty(gherkinDocument: messages.GherkinDocument, syntax: Syntax): string {
```

Comment:
Maybe we could add a default value 'gherkin' to `syntax` to avoid breaking changes to the interface?

Reply:
It's an internal library, so I'm not worried about breaking the interface. I always avoid default argument values unless it's for keeping backwards compatibility (in public interfaces).

Reply:
The library is published on npm publicly, and the pretty method is publicly exposed
Even if it is not documented, some developer may use it so it remains a breaking change of the public API IMO

Reply:
Done - defaults to 'gherkin'